metadatastore/test/test_sample_data.py

```diff
@@ -1,42 +1,39 @@
 from metadatastore.examples.sample_data.common import apply_deadband, noisy
 from metadatastore.examples.sample_data import (temperature_ramp,
                                                 multisource_event)
-from metadatastore.utils.testing import mds_setup, mds_teardown
+from .utils import mds_setup, mds_teardown
 import numpy as np
 import itertools
-from nose.tools import raises
+import pytest
 
-def setup():
+
+def setup_function(function):
```

Comment:
do we really want to do this on a per-function not per-module basis?  Saw playing with data broker that this majorly slows down the tests.